src/chrome/content/rules/Blic

```diff
@@ -0,0 +1,21 @@
+<!--
+	Invalid certificate:
+		s.blic.rs
+	Unable to connect to server:
+		recepti.zena.blic.rs
```

Comment:
`lavalbum.blic.rs`

Reply:
https://travis-ci.org/EFForg/https-everywhere/jobs/587991443#L327
Not exist.